src/utils/LogicUtils.res

```diff
@@ -233,6 +233,10 @@ let getOptionIntFromString = str => {
   str->Int.fromString
 }
 
+let getOptionStringFromInt = int => {
+  Some(int->Option.getOr(0)->Int.toString)
+}
+
```

Comment:
why it is always returning Some? If it is option string then why it is never returning none ?

Reply:
updated